articles/vmware/vmw-sco-dedicated-compute.md

```diff
@@ -59,11 +60,11 @@ During a failure scenario, VMs will be restarted on surviving hosts within the D
 
 The table below shows the configuration, the resources available to a customer, the number of physical blades in the configuration, and the UKCloud resources reserved to run it.
 
-&nbsp; | Customer available resources | Physical blades | Resource reservation
-------|------------------------------|-----------------|---------------------
-**Starter pack** | 20 cores, 300 GiB RAM | 2 | 50% (20 cores, 300 GiB RAM)
-**Starter pack + 1 expansion pack** | 40 cores, 600 GiB RAM | 3 | 33% (20 cores, 300 GiB RAM)
-**Starter pack + 2 expansion packs** | 60 cores, 900 GiB RAM | 4 | 25% (20 cores, 300 GiB RAM)
+&nbsp;                               | Customer available resources | Physical blades | Resource reservation
+-------------------------------------|------------------------------|-----------------|---------------------
+**Starter pack**                     | 20 cores, 300 GiB RAM        | 2               | 50% (20 cores, 300 GiB RAM)
```

Comment:
Are we sure they are 20 core? The price guide for Dedicated v1 states 16 cores (2 * 8) - unless this document is right, and the price guide is wrong. In which case, we need to change the price guide.